umbraco/Umbraco.UI
# Inline font files that were missing

## Description

This fixes a bug where font files were not being loaded correctly in the finalized uui-css bundle. Additionally we are adding some missing documentation to README and Storybook including fixes old names of files and being much clearer on how to use the uui-css package.

Fixes [AB#16713](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/16713)

## Types of changes




## Motivation and context

Before the path to the font-files were not correct in the `@font-face` declarations but this has been solved using the package `postcss-url` and its' default rebase mode.

## How to test?

1. Check Storybook that fonts are still included
2. Build the CSS project and make an example index.html file and include the compiled bundles there and check that the font loads by setting a \<h1> tag or something

## Screenshots (if appropriate)

<img width="1136" alt="image" src="https://user-images.githubusercontent.com/752371/153603650-77cf48c9-1024-4844-a076-57787023cc53.png">


## Checklist
